inference-engine/thirdparty/clDNN/kernel_selector/core/kernel_selector_params.h

```diff
@@ -488,6 +488,19 @@ struct FusedOpsConfiguration {
     bool IsPostReorderFused(void) const { return orig_output_layout != DataLayout::DataLayoutCount; }
 };
 
+enum class Deptype {
```

Comment:
Please add a description here what this Deptype is used for and what does each enum value mean. Also for the naming convention it should be DepType

Reply:
Fixed it. thanks.